R/task-eval.R

```diff
@@ -38,7 +38,7 @@ task_eval <- function(id, envir = .GlobalEnv, verbose = FALSE, root = NULL) {
     version <- utils::packageVersion("hipercow")
     cli::cli_h1("hipercow {version} running at '{root$path$root}'")
     cli::cli_alert_info("library paths:")
-    cli::cli_li(.libPaths())
+    cli::cli_ul(.libPaths())
```

Comment:
`cli_li` was leaving a container open, which then indented all other bullet points in the function.
`cli_ul` does not have this problem, as it closes the container when given a non-NULL argument.